src/ErrorWrapper.js

```diff
@@ -30,7 +33,7 @@ function ErrorBoundary({ onError }) {
 	}, []);
 
 	return (
-		<Route path="/" children={<App/>}/>
+		<App history={history} />
```

Comment:
We doesn't need a router here because the `children` API from `Route` renders the component either it matches the path or not. Therefore, we can just render the component directly.

In general, the `children` API is useful for animation or style.

See [documentation](https://v5.reactrouter.com/web/api/Route/children-func).